services/api/handlers/projects.js

```diff
@@ -254,6 +254,27 @@ exports.patch = {
   }
 };
 
+exports.delShallow = function(request, reply) {
+  request.server.methods.projects.remove(
+    [
+      request.params.project
+    ],
+    function(err, result) {
+      if(err) {
```

Comment:
coveralls is upset about this line not being tested, but it *is*, here: https://github.com/mozilla/api.webmaker.org/blob/develop/test/services/api/handlers/projects.js#L1835-L1848